dailymotion.py

```diff
@@ -249,6 +260,9 @@ def oauth_token_request(self, params):
     def get_access_token(self, force_refresh=False, request_args=None):
         params = {}
 
+        if self.access_token is not None:
```

Comment:
The behavior behind "force refresh" will be broken if the access token is previously set. Is it the aim ?